Cargo.toml

```diff
@@ -21,6 +21,9 @@ env_logger = "0.7.1"
 kubelet = { path = "./crates/kubelet", version = "0.1.0", features = ["cli"] }
 wascc-provider = { path = "./crates/wascc-provider", version = "0.1.0" }
 wasi-provider = { path = "./crates/wasi-provider", version = "0.1.0" }
+serde_json = "1.0"
```

Comment:
These should be added under `dev-dependencies`. Also, I think that it is better to add the `features = ["derive"]` to the `serde` dependency instead of importing `serde_derive` directly

Reply:
> the package `krustlet` depends on `serde_json`, with features: `derive` but `serde_json` does not have these features.

Reverting back to previous code. Note: this is the same as what's in `crates/kubelet/Cargo.toml`. Let's revisit later

Reply:
Yeah, I meant that you'd need to bring in `serde` and `serde_json`